packages/server-wallet/.env.test

```diff
@@ -6,6 +6,6 @@ SERVER_DB_USER=root
 HUB_DESTINATION='hub'
 
 GANACHE_HOST=0.0.0.0
-GANACHE_PORT=8547
+GANACHE_PORT=8545
```

Comment:
Is this to avoid the polling issue? Maybe we could leave a comment to clarify :)

Reply:
The reason I made this change is because other places in the `server-wallet` reference port `8545` like [here](https://github.com/statechannels/statechannels/blob/chain-service/packages/server-wallet/package.json#L95) and [here](https://github.com/statechannels/statechannels/blob/chain-service/packages/server-wallet/deployment/deploy.ts#L18). We should probably change both of those to reference an environment variable instead of hardcoding `8545`.